packages/x-charts/src/PieChart/PieArc.tsx

```diff
@@ -105,22 +183,18 @@ function PieArc(props: PieArcProps) {
   const classes = useUtilityClasses(ownerState);
 
   const interactionProps = useInteractionItemProps({ type: 'pie', seriesId: id, dataIndex });
+  const animatedProps = useAnimatePieArc({
+    cornerRadius,
+    startAngle,
+    endAngle,
+    innerRadius,
+    outerRadius,
+    paddingAngle,
+    skipAnimation,
+  });
 
   return (
     <PieArcRoot
-      d={to(
-        [startAngle, endAngle, paddingAngle, innerRadius, outerRadius, cornerRadius],
-        (sA, eA, pA, iR, oR, cR) =>
-          d3Arc().cornerRadius(cR)({
-            padAngle: pA,
-            startAngle: sA,
-            endAngle: eA,
-            innerRadius: iR,
-            outerRadius: oR,
-          })!,
-      )}
-      visibility={to([startAngle, endAngle], (sA, eA) => (sA === eA ? 'hidden' : 'visible'))}
```

Comment:
Is there a reason for this removed line? IIRC this was done to prevent a bug where a line was shown when the slice had a value of 0 or very small

Reply:
I can't reproduce this issue. Do you know how to? 

https://github.com/user-attachments/assets/924fdaf3-8e79-4917-9d0a-383c91393fac

Reply:
![Screenshot 2025-03-21 at 15 38 20](https://github.com/user-attachments/assets/c9154a17-412d-4732-921c-3049c0323589)

Reply:
https://github.com/mui/mui-x/pull/13897

Reply:
You're right, my change regresses this. Pushed a commit to fix it.